api/conftest.py

```diff
@@ -292,19 +292,36 @@ def environment_api_key(environment):
 
 
 @pytest.fixture()
-def master_api_key(organisation) -> Tuple[MasterAPIKey, str]:
-    master_api_key, key = MasterAPIKey.objects.create_key(
+def admin_master_api_key(organisation) -> typing.Tuple[MasterAPIKey, str]:
+    master_api_key, _ = MasterAPIKey.objects.create_key(
+        name="test_key", organisation=organisation, is_admin=True
+    )
+    return master_api_key
+
+
+@pytest.fixture()
+def master_api_key(organisation) -> typing.Tuple[MasterAPIKey, str]:
+    master_api_key, _ = MasterAPIKey.objects.create_key(
+        name="test_key", organisation=organisation, is_admin=False
+    )
+    return master_api_key
+
+
+@pytest.fixture()
+def master_api_key_and_obj(organisation) -> typing.Tuple[MasterAPIKey, str]:
+    master_api_key_obj, key = MasterAPIKey.objects.create_key(
         name="test_key", organisation=organisation
     )
-    return master_api_key, key
+    return key, master_api_key_obj
 
 
 @pytest.fixture()
-def master_api_key_client(master_api_key):
+def master_api_key_client(master_api_key_and_obj):
+    key = master_api_key_and_obj[0]
     # Can not use `api_client` fixture here because:
     # https://docs.pytest.org/en/6.2.x/fixture.html#fixtures-can-be-requested-more-than-once-per-test-return-values-are-cached
     api_client = APIClient()
-    api_client.credentials(HTTP_AUTHORIZATION="Api-Key " + master_api_key[1])
+    api_client.credentials(HTTP_AUTHORIZATION="Api-Key " + key)
```

Comment:
Some thoughts here: 

 1. The arguments to the fixtures should have type hints. 
 2. `master_api_key_and_obj` is a little odd, it also doesn't indicate whether it's an admin key or not. It feels like all these fixtures should just return the object itself and the key? Then we'd only have a need for 2 fixtures, right?

Reply:
Yeah, that's a fair point - I have added another _object fixture